test/js/pageobjects/Welcome.js

```diff
@@ -44,6 +45,14 @@ class Welcome extends Base {
   async clickOnCloseModalButton() {
     this.closeModalButton.click()
   }
+
+  async checkBackArrowIsNotDisplayed() {
```

Comment:
We should have this method also on the Complete screen

Reply:
Added this to one of the tests in 78f828f